### PR TITLE
SDK-754: Introduced naming and console command structure conventions.

### PR DESCRIPTION
**Pull request by pavelmaksimov25**
Dear colleagues. 
I've provided additional conventions. Any constructive ideas, objections, and improvements are more than welcome!

Ticket: https://spryker.atlassian.net/browse/SDK-754

### CODE REVIEW

File: docs/conventions.md [hunk 1]
```diff
@@ -35,3 +35,39 @@ No conventions yet
 - name __MUST__ start and end with `%` (e.g.: `%some_placeholder%`)
 - __MUST__ define `optional: true/false` to indicate if the placeholders needs to be resolved to run the task
 - __MUST__ use the `id` or full qualified class name of an existing `ValueResolver` for the field `valueResolver`
+
+## Naming
+
+### Folder naming
+
+- Folder's name __MUST__ be plural. Event instead of Events.
```
Comment (dereuromark) [inline docs/conventions.md:43]: Example does not much the rule.

File: docs/conventions.md [hunk 2]
```diff
@@ -35,3 +35,39 @@ No conventions yet
...
+- Folder's name __MUST__ have a StudlyCaps(PascalCase) format.
```
Comment (dereuromark) [inline docs/conventions.md:44]: Not needed as this is defined as per PSR and company wide policy

File: docs/conventions.md [hunk 3]
```diff
@@ -35,3 +35,39 @@ No conventions yet
...
+
+### Class naming
+
+- The name of the class __MUST__ be plural. `AcmeTask` instead of `AcmeTasks`.
```
Comment (dereuromark) [inline docs/conventions.md:48]: Same example mismatch

File: docs/conventions.md [hunk 4]
```diff
@@ -35,3 +35,39 @@ No conventions yet
...
+- Class's name __MUST__ have a StudlyCaps(PascalCase) format.
```
Comment (dereuromark) [inline docs/conventions.md:49]: Again PSR, remove IMO

**Review by andrea-taini — APPROVED** [review]

**Review by Kyrylo-Meretskyi — DISMISSED** [review]

**Pull request merged**